src/components/React/Howto.js

```diff
@@ -25,96 +25,89 @@ export default function Howto(props) {
+              properties of the data. Some regions will highlight more than
+              others according to specific colors.
             </p>
             <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/channels.gif"
-              alt="Instructional gif on selecting RGBA channels"
+              src="assets/images/howto/riddc-color-map-guide.gif"
+              alt="Instructional gif on applying a color texture"
               width="100%"
             />
-            <hr className="my-4" />
           </Row>
-
-          <Row>
+          <Row className="my-4">
             <p>
-              Check the “Enable color mapping” box to apply a 1d color texture
-              to the dataset. Some regions will highlight more than others
-              according to specific colors. Use the Transfer function graph to
-              modify the transparency of pixels according to their mapped color.
-              To add point double click on the place you want to put a new
-              control point. Right click on an existing point to delete it.
+              Use the Transfer function graph to modify the transparency of
+              pixels according to their mapped color. This allows you to
+              highlight/obscure specific ranges of data. To add point double
+              click on the place you want to put a new control point. Right
+              click on an existing point to delete it.
             </p>
             <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/colorMappingTransfer.gif"
+              src="assets/images/howto/riddc-opacity-map-guide.gif"
               alt="Instructional gif on applying a color texture"
               width="100%"
-            ></img>
-            <hr className="my-4" />
+            />
           </Row>
-          <Row>
+          <Row className="my-4">
             <p>
-              Lastly, use the ranged slices to cut through the dataset on three
-              different axes.
+              Lastly, to clip the data, use the ranged slices to cut through the
+              dataset on three different axes.
             </p>
             <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/slice.gif"
+              src="assets/images/howto/riddc-slices-guide.gif"
               alt="Instructional gif on cutting through the dataset on an axis"
               width="100%"
             />
-            <hr className="my-4" />
           </Row>
+          <hr className="my-4" />
 
-          <h2 id="enabling-vr" className="mb-4">
-            Enable VR on Web Browser
-          </h2>
-          <Row>
+          <h2 id="enabling-vr">Enable VR Headset on Web Browser</h2>
+          <Row className="my-4">
             <p>
               At this moment, VR on web browsers is only supported on Mozilla
```

Comment:
add prerequisites for VR in the section (nvidia, only tested on firefox, maybe only windows?)

Reply:
Done. Please check again.

Reply:
I changed to:

e.g. Oculus, Vive

Please check again.